soc/raspberrypi/soc.yml

```diff
@@ -1,6 +1,9 @@
 family:
 - name: rpi_pico
   series:
-  - name: rp2xxx
+  - name: rp20xx
     socs:
     - name: rp2040
+  - name: rp23xx
+    socs:
+    - name: rp2350
```

Comment:
This line should be `rp2350a`

Reply:
Yes, noted already.

I'm trying to ensure that we have a set of self-contained logical commits, so this requires a bit of careful `fixup!` and rebasing on my end: I don't want to pollute `main` with a load of fettling commits.